hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/main/java/org/apache/hadoop/yarn/server/router/Router.java

```diff
@@ -209,7 +214,48 @@ public void startWepApp() {
 
     Builder<Object> builder =
         WebApps.$for("cluster", null, null, "ws").with(conf).at(webAppAddress);
-    webApp = builder.start(new RouterWebApp(this));
+
+    WebAppContext uiWebAppContext = null;
```

Comment:
I guess there's no way to add a test for this?
Did you try locally, can you add screenshot before and after at least?

Reply:
Thank you very much for your help in reviewing the code! I will provide screenshot information as soon as possible.  

YARN 3.0 already supports WebUI2, but our Router does not yet support this. This pr is the first pr that supports WebUI2, allowing users to use WebUI2 pages. After this pr is completed, we will provide 2-3 new pages about the router based on the framework of WebUI2.

Reply:
- Cluster Overview

![image](https://github.com/apache/hadoop/assets/55643692/11a710e5-b497-42bf-aa57-04c78b4d01cf)

- Queue Overview

![image](https://github.com/apache/hadoop/assets/55643692/832e0305-0a98-4051-8f62-30cec86d99e0)

- Node
![image](https://github.com/apache/hadoop/assets/55643692/aa9099da-7397-45bc-a20a-30f19cb0803e)